core/jazz_ui/e2e/common/commontest.ts

```diff
@@ -17,42 +17,52 @@ import { CONFIGURATIONS } from '../../src/config/configuration';
 const fs = require('fs');
 const config = CONFIGURATIONS.optional.general.e2e;
 import { Jazz } from '../page-objects/jazzservices.po';
+import { Login } from '../page-objects/login.po';
 
 let servicename;
-const timeOutHigh = 180000;
-const EC = protractor.ExpectedConditions;
-const twok = 2000;
-const fivek = 5000;
-const tenk = 10000;
-const fifteenk = 15000;
-const twentyk = 20000;
-const thirtyk = 30000;
-const sixtyk = 60000;
 let jazzServices_po: Jazz;
 jazzServices_po = new Jazz();
 
-export class Comman {
+let login_po: Login;
+login_po = new Login();
+
+
+export class Common {
+  public static readonly emailId = CONFIGURATIONS.optional.general.e2e.EMAIL_ID;
+  public static readonly config = CONFIGURATIONS.optional.general.e2e;
+  public static readonly timeOutHigh = 180000;
+  public static readonly EC = protractor.ExpectedConditions;
+  public static readonly twok = 2000;
```

Comment:
Need more logical names instead of the values that they signify. Something like longWait, shortWait, xLWait, etc.

Reply:
done